src/libs/map-engine.ts

```diff
@@ -269,28 +275,45 @@ export default class MapEngine {
 }
 
 const createOrUpdateAirportMarker =
-  (title: string, longitude: number, latitude: number, marker?: google.maps.Marker) => {
-    marker = marker || new google.maps.Marker({
+  (title: string, longitude: number, latitude: number, markerData: AirportMarkerData): AirportMarkerData => {
+    markerData.marker = markerData.marker || new google.maps.Marker({
         draggable: false,
         optimized: true,
         title,
     });
-    marker.setPosition({lat: latitude, lng: longitude});
-    return marker;
+    markerData.marker.setPosition({lat: latitude, lng: longitude});
+    drawAirportMarker(markerData);
+    return markerData;
 };
 
-const createOrUpdateMarker = (icaoNumber: string, longitude: number, latitude: number, markerData: MarkerData) => {
+function drawAirportMarker(markerData: AirportMarkerData): void {
+  const innerColor = markerData.enabled ? '#933400' : '#7a7a7a';
```

Comment:
I'll use the light orange here `#FB8F2D`

Reply:
I rather disagree here. A lighter orange tends to disappear when close to a cluster of darker bordered icons.